rero_ils/modules/items/serializers/csv.py

```diff
@@ -30,12 +30,14 @@
 from rero_ils.modules.serializers import CachedDataSerializerMixin
 from rero_ils.utils import get_i18n_supported_languages
 
-from .collector import Collecter
+from .collector import Collector
 
 
 class ItemCSVSerializer(CSVSerializer, CachedDataSerializerMixin):
     """Serialize item search for csv."""
 
+    collector = Collector
```

Comment:
Why ? What's the purpose of store the `Collector` class reference as `self.collector`. Simply use `Collector.my_requested_method()`